frontend/e2e/app.spec.ts

```
@@ -0,0 +1,166 @@
+  await urlInput.fill("https://httpbin.org/get");
+
+  // Click send button
+  const sendButton = page.getByRole("button", {name: "Send"});
+  await sendButton.click();
+
+  // Wait for response
+  await page.waitForSelector("text=200", {timeout: 10000});
+
+  // Verify response appears
+  const responseText = await page.textContent("body");
+  expect(responseText).toContain("200");
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)